### PR TITLE
ICSP and IDMS interoperability release note 4.13

### PR DESCRIPTION
**Pull request by mburke5678**
Release note for https://github.com/openshift/openshift-docs/pull/70881

Issues:
https://issues.redhat.com//browse/OCPBUGS-22275
https://issues.redhat.com/browse/OSDOCS-9304

Preview: https://72316--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes#ocp-4-13-27-bug-fixes

[Change management discussion](https://redhat-internal.slack.com/archives/C06LX2S9NDU/p1709142556910039) in Slack.

Text is the same as https://github.com/openshift/openshift-docs/pull/72317

### CODE REVIEW

**Review by QiWang19 — COMMENTED** [review]
LGTM

**Comment by rphillips** [comment]
Let's move this section to the Bug Fixes heading.

**Review by QiWang19 — COMMENTED** [review]
LGTM

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -3713,6 +3713,16 @@ You can view the container images in this release by running the following comma
 $ oc adm release info 4.13.27 --pullspecs
 ----
 
+[id="ocp-4-13-27-bug-fixes"]
+==== Bug fixes
+
+* Previously, `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS) objects could not be used if there were any `ImageContentSourcePolicy` (ICSP) objects in the cluster. As a consequence, to use IDMS or ITMS objects, you needed to delete any ICSP objects in the cluster, which required a cluster reboot. With this change, ICSP, IDMS, and ITMS objects now function in the same cluster at the same time. As a result, you can use any or all of the three types of objects to configure repository mirroring after the cluster is installed. For more information, see xref:../post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror_post-install-preparing-for-users[Understanding image registry repository mirroring]. (link:https://issues.redhat.com/browse/RHIBMCS-185[RHIBMCS-185])
```
Comment (bergerhoffer) [inline release_notes/ocp-4-13-release-notes.adoc:3719]: A few suggestions to consider

```suggestion
* Previously, `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS) objects could not be used if there were any `ImageContentSourcePolicy` (ICSP) objects in the cluster. As a result, to use IDMS or ITMS objects, you needed to delete any ICSP objects in the cluster, which required a cluster reboot. With this release, ICSP, IDMS, and ITMS objects now function in the same cluster at the same time. As a result, you can use any or all of the three types of objects to configure repository mirroring after the cluster is installed. For more information, see xref:../post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror_post-install-preparing-for-users[Understanding image registry repository mirroring]. (link:https://issues.redhat.com/browse/RHIBMCS-185[RHIBMCS-185])
```

**Review by bergerhoffer — COMMENTED** [review]
Minor suggestions

**Comment by xltian** [comment]
lgtm

**Pull request merged**